Marlin/Configuration.h

```diff
@@ -729,9 +729,20 @@
 #define DEFAULT_RETRACT_ACCELERATION  3000    // E acceleration for retracts
 #define DEFAULT_TRAVEL_ACCELERATION   3000    // X, Y, Z acceleration for travel (non printing) moves
 
-//
-// Use Junction Deviation instead of traditional Jerk Limiting
-//
+/**
+ * Junction Deviation
+ *  
+ * Use Junction Deviation instead of traditional Jerk Limiting
+ * 
+ * A = DEFAULT_ACCELERATION (acceleration for printing moves)
+ * V = Jerk for X and Y (values typically match)
+ * 
+ * Junction Deviation = 0.4*A^2/V
+ * 
+ * Ex: If your Jerk value is 8.0 and your acceleration is 1250, then: 0.4*8.0^2/1250 = 0.02048 (mm)
```

Comment:
This will need to be clarified, since when `JUNCTION_DEVIATION` is enabled, Marlin has no Jerk values. You cannot set XYZ Jerk, and the Jerk values set in the configuration do not apply. So, where does the 8.0 value come from?

Reply:
> So, where does the "jerk value" of 8.0 come from?

I wasn't sure how to approach that issue since `JUNCTION_DEVIATION` is not enabled by default and Jerk settings are still being used/commonly available for most printers, simplifying conversion.

Are there plans to move away from Jerk settings soon/for Marlin 2.0?

Reply:
First priority might be to decide:

```
* Junction Deviation = 0.4*A^2/V
 * 
 * Ex: If your Jerk value is 8.0 and your acceleration is 1250, then: 0.4*8.0^2/1250 = 0.02048 (mm)
```

Isn't it `0.4*V²/A` ?

Reply:
You could be given a choice:

1. Don't set a JD value but set archaic jerk values ==> archaic jerk is used with these values

2. Set a JD value of `0` and set archaic jerk values ==> JD is used and Marlin translates the jerk value(s) via the formula

3. Set a JD value ( `<> 0`) and JD will be used with that value.

Something like that, with a little explanation in the comments leading up to those #defines

Reply:
> Isn't it `0.4*V²/A` ?

It is. Good catch.

I'm open to how it should be worded/done, but wording will depend on what the plans are for traditional Jerk settings.